### PR TITLE
Add Jenkinsfile to push docker image and helm chart

### PR DESCRIPTION
**Pull request by kd7lxl**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -55,8 +55,7 @@ build-example: fmt bin build
 
 # formats the repo
 fmt:
-	@echo "Running 'go fmt ...'"
-	@go fmt -x "$(REPO)/..."
```
Comment (kd7lxl) [inline Makefile:59]: This is the only place that depended on `go` outside of a container. Moving it into the container eliminates the dependency on `go` on CI nodes.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    stage("Build Image") {
+      steps {
+        sh 'make build image VERSION=$TAG'
+      }
+    }
+    stage("Package Chart") {
+      steps {
+        withAWS(credentials: "CICD_HELM", region: "us-east-1") {
+          sh '''
+            docker run --rm \
```
Comment (Calebjh) [inline Jenkinsfile:27]: Do we need to rm this chart after it is pushed, or will it be automatically cleaned up?

**Review by Calebjh — APPROVED** [review]
I suppose if we cleanup the chart we wouldn't be able to archive it as an artifact...

File: Jenkinsfile (hunk 2 above)
Comment (kd7lxl) [reply Jenkinsfile:27]: The Jenkins nodes are ephemeral, so I don't think cleanup is a big deal.

**Pull request merged**